<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(233, 213, 218);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(172, 125, 136);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="text">
       <string>==OVERALL ACTIVITY==</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="txtCreate">
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Account created today</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="txtWithdraw">
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Money withdrawn today</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="txtDeposit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Money deposited today</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="txtChange">
      <property name="font">
       <font>
        <family>Bahnschrift SemiCondensed</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Account change</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(172, 125, 136);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
